doc-content/enterprise-only/getting-started-kogito/con-kogito-probes.adoc

```diff
@@ -0,0 +1,4 @@
+[id="con-kogito-probes_{context}"]
+= Probes for {KOGITO} microservices on {OPENSHIFT}
+
+The probes in {OPENSHIFT} verify that an application is working or it needs to be restarted. For {KOGITO} microservices on Quarkus and Spring Boot, probes interact with the application using an HTTP request, defaulting to the endpoints that are exposed by an extension. Therefore, to run your {KOGITO} microservices on {OPENSHIFT}, you must import the extensions to enable the https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes[liveness, readiness, and startup probes] for your application.
```

Comment:
` Therefore, to run your {KOGITO} microservices on {OPENSHIFT}, you must import the extensions` This is valid just for Quarkus (for 7.11, in 7.12 it will be valid for SpringBoot too), should be mentioned there.